src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,114 @@
+.product-card {
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  &__image {
+    padding-inline: 3px;
+
+    & img {
+      width: 100%;
+      height: 134px;
+      background-size: cover;
+    }
+  }
+
+  &__title {
+    margin: 0;
+    font-family: inherit;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color-main;
+  }
+
+  &__code {
+    margin: 4px 0 16px;
+    font-family: inherit;
+    font-size: 10px;
+    line-height: 14px;
+    color: rgba(97, 96, 112, 1);
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__stars {
+    display: flex;
+  }
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url('/src/images/star-active.svg');
+  }
```

Comment:
This way of setting stars is not flexible. The number of active stars should be set dynamically, based on the product rating. Consider using a loop to set the active stars, or move this logic to the HTML or JS, if you have such.